src/shared/schemas.ts

```diff
@@ -17,8 +14,12 @@ import { Settings } from './settings'
 import { once } from './utilities/functionUtils'
 import { normalizeSeparator } from './utilities/pathUtils'
 import { Any, ArrayConstructor } from './utilities/typeConstructors'
+import { AWS_SCHEME } from './constants'
+import { writeFile } from 'fs-extra'
+import { SystemUtilities } from './systemUtilities'
 
 const GOFORMATION_MANIFEST_URL = 'https://api.github.com/repos/awslabs/goformation/releases/latest'
+const SCHEMA_PREFIX = `${AWS_SCHEME}://aws-toolkit-vscode/`
```

Comment:
* The `aws-toolkit-vscode/` is not necessary because we "own" all `aws*://` schemes. Does it serve a technical or UX purpose? If not, let's drop it. 
* To disambiguate other Toolkit urls for s3, cwl, etc., our current approach is to claim `aws-s3://` and `aws-cwl://` schemes. However I see below we're doing `SCHEMA_PREFIX + sam.schema.json` to disambiguate, so that should be fine.